docs/extensions/reference/_posts/1970-01-06-CLI.md

```diff
@@ -296,18 +296,17 @@ Linked extensions:
 
 - ##### shoutem run [\<app-id>] [\--dev] [\--local] [\--small]
 
-Executes React Native packager locally to **run** and **debug** custom or existing extensions using **Shoutem Preview** app (available for [iOS]({{ site.shoutem.previewAppiOS }}) and [Android]({{ site.shoutem.previewAppAndroid }})) on both Android and iOS without having to setup Android Studio and/or XCode. Moreover, you can use Windows to develop iOS apps. The only 2 things you would need to run the app are `react-native-cli` and `Node.js` v7. See our [FAQ]({{ site.url }}/docs/extensions/tutorials/faq) if you have problems setting this up.
+Executes React Native packager locally to **run** and **debug** react-native apps using **Shoutem Preview** app (available for [iOS]({{ site.shoutem.previewAppiOS }}) and [Android]({{ site.shoutem.previewAppAndroid }})) on both Android and iOS without having to setup Android Studio and/or XCode. Moreover, you can use Windows to develop iOS apps. The only requirement for running the app is `Node.js` v6 or higher. See our [FAQ]({{ site.url }}/docs/extensions/tutorials/faq) if you have problems setting this up.
+
+Just change directory to a shoutem (or any react-native) app and run `shoutem run`.
 
 ```ShellSession
 $ shoutem run
-Select your app: Restaurants ({{ site.example.appId }})
```

Comment:
Isn't the idea that if you run it inside of an app directory after doing `shoutem clone` that you get that app specifically, since you're in that app's directory?